config/cluster/host.go

```diff
@@ -379,6 +379,22 @@ func (h *Host) NeedIPTables() bool {
 	return !h.Configurer.CommandExist(h, "iptables")
 }
 
+// NeedInetUtils returns true when the inetutils package is needed on the host to run `hostname`.
+// Placing this check here is kind of meh because this package probably only exists on arch.
```

Comment:
I think the `inetutils` package exists on most distros but it'spre-installed on almost all of them. Installing it when `hostname` is missing may actually solve some rare issues for other distros.